README.md

```diff
@@ -37,15 +37,37 @@ npm test
 
 ## Usage
 
-### Add/Update a Verifiable Credential
-
-Please add or update credentials in the `./credenitals` dir. When adding new
-credential, make sure the directory name is camel-cased. The directory should
-contain a logo image (with `.jpg` or `.png` file extensions) and the credential
-in the `credential.json` file. Additionally, you can also add a file for the
-credential context in the `./contexts` dir (for example see -
-`./contexts/alumni/v1.json`) if you want to host the contexts in
-[`@credential-handler/vc-playground-contexts-server`](https://github.com/credential-handler/vc-playground-contexts-server).
+### Add / Update a Verifiable Credential
+
+To add or update credentials in the `./credentials` directory:
+
+1. Within the `./credentials` directory, create a new directory for the
+credential. Make sure to use camel case for the directory name.
+2. Inside the newly created credential directory, add a `credential.json` file.
+This file should contain all the necessary information for the credential,
+including the issuer, credential subject, and other relevant data.
+3. Additionally, place a logo `image` file for the credential in the
+corresponding directory. The image file should be in either `JPG` or `PNG`
+format.
+
+### Add / Update a Credential Context
+
+To optionally add or update contexts in the `./contexts` directory:
+
+1. If you wish to include a context for a credential, create a new directory
+for the context in the `./contexts` directory. Make sure to use snake case for
```

Comment:
[Snake case](https://en.wikipedia.org/wiki/Letter_case#Snake_case) uses underscores, this should be [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case).  May want links for these terms.

Reply:
Good catch, idk what I was thinking :)

Reply:
Fixed [40aaaa2](https://github.com/credential-handler/vc-examples/pull/15/commits/40aaaa29ff59686dcfb3f8336092f8661e6eab42)